pymatgen/analysis/phase_diagram.py

```diff
@@ -449,18 +449,6 @@ def stable_entries(self):
         """
         return self._stable_entries
 
-    @lru_cache(2)  # cache in case of repeated calls
-    def get_stable_entries_normed(self, mode="formula_unit"):
-        """
-        Args:
-            mode (str): type of normalization to perform.
-                Allowed ["atom", "formula_unit"]
-
-        Returns:
-            list of normalized stable entries in the phase diagram.
-        """
-        return [e.normalize(mode) for e in self._stable_entries]
-
```

Comment:
I introduced this in order to get this to work before the changes to `__hash__` and how this method deals with polymorphs therefore in order to keep the code clean as I do not see other workflows method currently I have taken it out.